Index: build.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/build.gradle b/build.gradle
--- a/build.gradle	(revision 0af818d3f2b8a15b97351ef5464e3da5a527f157)
+++ b/build.gradle	(date 1733222424998)
@@ -1,11 +1,14 @@
+import java.util.regex.Pattern
+
 group 'com.grafana'
 version '10.0.0-SNAPSHOT' // the version of the actual release is set during the release process in build-release.sh
 
 buildscript {
   ext {
-    otelInstrumentationVersion = "2.10.0"
+    otelInstrumentationVersion = "2.11.0-SNAPSHOT"
   }
   repositories {
+    mavenLocal()
     maven {
       url "https://plugins.gradle.org/m2/"
     }
@@ -13,7 +16,7 @@
   dependencies {
     classpath "com.diffplug.spotless:spotless-plugin-gradle:6.25.0"
     classpath "com.gradleup.shadow:shadow-gradle-plugin:8.3.5"
-    classpath "io.opentelemetry.instrumentation:gradle-plugins:${otelInstrumentationVersion}-alpha"
+    classpath "io.opentelemetry.instrumentation:gradle-plugins:2.10.0-alpha"
   }
 }
 
@@ -26,7 +29,7 @@
   ext {
     versions = [
       opentelemetryJavaagent     : otelInstrumentationVersion,
-      opentelemetryJavaagentAlpha: "${otelInstrumentationVersion}-alpha",
+      opentelemetryJavaagentAlpha: otelInstrumentationVersion.replaceFirst(Pattern.compile("(-SNAPSHOT)?\$"), "-alpha\$1"),
 
       bytebuddy                  : "1.15.10",
       autoservice                : "1.1.1",
@@ -45,6 +48,7 @@
   }
 
   repositories {
+    mavenLocal()
     mavenCentral()
   }
 
