Index: examples/spring-boot-demo/build.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/examples/spring-boot-demo/build.gradle b/examples/spring-boot-demo/build.gradle
--- a/examples/spring-boot-demo/build.gradle	(revision 95877a76813f08c423b225716d6a4346274aecb0)
+++ b/examples/spring-boot-demo/build.gradle	(date 1686646608894)
@@ -1,12 +1,12 @@
 plugins {
 	id 'java'
-	id 'org.springframework.boot' version '3.1.0'
-	id 'io.spring.dependency-management' version '1.1.0'
+	id 'org.springframework.boot' version '2.7.12'
+	id 'io.spring.dependency-management' version '1.0.15.RELEASE'
 }

 group = 'com.grafana'
 version = '0.0.1-SNAPSHOT'
-sourceCompatibility = '17'
+sourceCompatibility = '1.8'

 repositories {
 	mavenCentral()
@@ -18,7 +18,7 @@
 	implementation 'redis.clients:jedis:4.4.1'
 	implementation 'org.springframework.boot:spring-boot-starter-data-mongodb'
 	implementation 'org.springframework.kafka:spring-kafka'
-	implementation 'com.ibm.mq:mq-jms-spring-boot-starter:3.1.0'
+	implementation 'com.ibm.mq:mq-jms-spring-boot-starter:2.7.12'
 	implementation 'org.springframework.boot:spring-boot-starter-web'
 	implementation 'org.springframework.boot:spring-boot-starter-actuator'
 	implementation 'io.micrometer:micrometer-registry-prometheus' //only for debugging
Index: examples/spring-boot-demo/src/main/java/com/grafana/springbootdemo/Product.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/examples/spring-boot-demo/src/main/java/com/grafana/springbootdemo/Product.java b/examples/spring-boot-demo/src/main/java/com/grafana/springbootdemo/Product.java
--- a/examples/spring-boot-demo/src/main/java/com/grafana/springbootdemo/Product.java	(revision 95877a76813f08c423b225716d6a4346274aecb0)
+++ b/examples/spring-boot-demo/src/main/java/com/grafana/springbootdemo/Product.java	(date 1686646840561)
@@ -1,6 +1,6 @@
 package com.grafana.springbootdemo;

-import jakarta.persistence.*;
+import javax.persistence.*;

 @Entity
 public class Product {
Index: examples/spring-boot-demo/.tool-versions
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/examples/spring-boot-demo/.tool-versions b/examples/spring-boot-demo/.tool-versions
--- a/examples/spring-boot-demo/.tool-versions	(revision 95877a76813f08c423b225716d6a4346274aecb0)
+++ b/examples/spring-boot-demo/.tool-versions	(date 1686647284292)
@@ -1,1 +1,1 @@
-java temurin-19.0.2+7
+java temurin-8.0.372+7
Index: examples/spring-boot-demo/src/main/java/com/grafana/springbootdemo/JmsConfig.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/examples/spring-boot-demo/src/main/java/com/grafana/springbootdemo/JmsConfig.java b/examples/spring-boot-demo/src/main/java/com/grafana/springbootdemo/JmsConfig.java
--- a/examples/spring-boot-demo/src/main/java/com/grafana/springbootdemo/JmsConfig.java	(revision 95877a76813f08c423b225716d6a4346274aecb0)
+++ b/examples/spring-boot-demo/src/main/java/com/grafana/springbootdemo/JmsConfig.java	(date 1686646840549)
@@ -1,6 +1,5 @@
 package com.grafana.springbootdemo;

-import jakarta.jms.ConnectionFactory;
 import org.springframework.boot.autoconfigure.jms.DefaultJmsListenerContainerFactoryConfigurer;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
@@ -11,6 +10,8 @@
 import org.springframework.jms.support.converter.MessageConverter;
 import org.springframework.jms.support.converter.MessageType;

+import javax.jms.ConnectionFactory;
+
 @Configuration
 @EnableJms
 public class JmsConfig {
